draft-ietf-jsonpath-base.md

```diff
@@ -1000,7 +997,7 @@ as many times in the node list.
 The filter selector has the form `[?<expr>]`. It works via iterating over structured values, i.e. arrays and objects.
 
 ~~~~ abnf
-filter-selector    = "[?" boolean-expr "]"
+filter-selector    = "[?" S boolean-expr S "]"
```

Comment:
So no space between `[` and `?`?

Reply:
no ... might be an extension to the standard making no harm.